grouper/permissions.py

```diff
@@ -550,25 +536,22 @@ def create_request(session, user, group, permission, argument, reason):
     return request
 
 
-def get_pending_request_by_group(session, group):
-    """Load pending request for a particular group.
-
-    Args:
-        session(sqlalchemy.orm.session.Session): database session
-        group(models.Group): group in question
-
-    Returns:
-        list of models.PermissionRequest correspodning to open/pending requests
-        for this group.
-    """
+def get_pending_request_by_group(session: Session, group: Group) -> List[PermissionRequest]:
+    """Load pending request for a particular group."""
     return (
         session.query(PermissionRequest)
         .filter(PermissionRequest.status == "pending", PermissionRequest.group_id == group.id)
         .all()
     )
 
 
-def can_approve_request(session, request, owner, group_ids=None, owners_by_arg_by_perm=None):
+def can_approve_request(
+    session: Session,
+    request: PermissionRequest,
+    owner: User,
+    group_ids: Optional[Set[int]] = None,
+    owners_by_arg_by_perm: Optional[Dict[object, Dict[str, List[Group]]]] = None,
+) -> Set[int]:
```

Comment:
Nit: Mind throwing a quick docstring on this function? Don't think it's super clear from name + args what it actually does. (Admittedly it's fairly straightforward to look at the code, but reading English is faster :) )

Feel free to decline of course, particularly because the behavior here might actually be harder to explain in English than with the code

Reply:
Good idea. It also uncovered that it only needs to return a bool.